.github/workflows/testAndPublish.yml

```diff
@@ -35,6 +35,8 @@ env:
   VSCMD_SKIP_SENDTELEMETRY: 1
   # Cache details about available MSVC tooling for subsequent SCons invocations to the provided file.
   SCONS_CACHE_MSVC_CONFIG: ".scons_msvc_cache.json"
+  defaultRunner: 'windows-2025'
+  supportedSystemTestRunners: '["windows-2025", "windows-2022"]'
```

Comment:
```suggestion
  supportedRunners: '["windows-2025"]'
```

Reply:
I will go back to 2025 only when I"m sure that the caching restore works whenever we add new runners.